python/doc/examples/reliability_sensitivity/estimate_probability_form.ipynb

```diff
@@ -146,7 +146,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Create the event whose probability we want to estimate."
+    "Create the event we want to estimate the probability."
```

Comment:
```suggestion
    "Create the event whose probability we want to estimate."
```
Perhaps you mean "Create the event we want to estimate the probability *of*"? But the original phrase works better in my opinion.